backend/data/scrapers/payload.py

```diff
@@ -12,7 +12,7 @@
     "content-type": "application/json",
 }
 
-def create_payload(size, content_type):
+def create_payload(size, content_type, year):
```

Comment:
Should `year` here have a default value set to `LIVE_YEAR`?

Reply:
Yup you're right